stats.go

```diff
@@ -107,3 +128,29 @@ func (s *statsExporter) submitMetric(v *view.View, row *view.Row, metricName str
 		return fmt.Errorf("aggregation %T is not supported", v.Aggregation)
 	}
 }
+
+func calculatePercentile(percentile float64, buckets []float64, countPerBucket []int64) float64 {
```

Comment:
This is a complex method, can we please explain it to readers? What are its parameters, etc. It would also be nice to add one or two inline comments to explain those steps for newbies like myself and others who are unfamiliar with this.

Reply:
Sure, added some docs to explain what the method does and also the parameters.

Reply:
That's great. Thanks.